src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +36,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+            .filter(user1 -> user1.getAge() != null)
+            .filter(user1 -> user1.getAge() >= 30)
```

Comment:
```suggestion
            .filter(user1 -> user1.getAge() >= 30 && user1.getAge() <= 30)
```
와 같이 쓰면 filter 를 하나 덜 거칠수 있을꺼 같아요!